utils/getConfig.js

```diff
@@ -0,0 +1,13 @@
+import fs from "fs/promises";
+import { configPath } from "./getConfigPath.js";
+
+const getConfig = async () => {
+  try {
+    const data = await fs.readFile(configPath, "utf-8");
+    return JSON.parse(data);
+  } catch {
+    return {};
+  }
+};
+
+export { getConfig };
```

Comment:
As we already have imported the config on translateController and will work with that, then we wouldn't need this util, it can be removed